.github/workflows/platform.assignIssueToProject.yml

```diff
@@ -18,7 +15,7 @@ jobs:
     steps:
       - name: 'Add Bug to Bug Board'
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.PLATFORM_PROJECT_TOKEN }}
```

Comment:
I guess this should be updated also in line 44

Reply:
Well spotted.